File: DeJia/app/src/main/java/com/dejia/anju/api/base/BaseCallBackUploadApi.java
package com.dejia.anju.api.base;

import android.content.Context;

import com.lzy.okgo.model.HttpParams;

import java.util.Map;


public interface BaseCallBackUploadApi {
    void getCallBack(Context context, Map<String, Object> maps, HttpParams uploadPamas, BaseCallBackListener listener);
}

File: DeJia/app/src/main/java/com/dejia/anju/api/base/BaseCallBackUploadApi.kt
package com.dejia.anju.api.base

import android.content.Context
import com.lzy.okgo.model.HttpParams

interface BaseCallBackUploadApi {
    fun getCallBack(context: Context, maps: MutableMap<String, Any>, uploadPamas: HttpParams, listener: BaseCallBackListener<*>)
}